test/check/check.go

```diff
@@ -468,11 +520,51 @@ func checkForDaemonSet(name string) error {
 }
 
 func checkForSecret(name string) error {
-	return framework.Global.Client.Get(context.Background(), types.NamespacedName{Name: name, Namespace: components.Namespace}, &corev1.Secret{})
+	secret := corev1.Secret{}
+	err := framework.Global.Client.Get(context.Background(), types.NamespacedName{Name: name, Namespace: components.Namespace}, &secret)
+	if err != nil {
+		return err
+	}
+
+	err = checkRelationshipLabels(secret.GetLabels(), "Secret", name)
+	if err != nil {
+		return err
+	}
+
+	return nil
 }
 
 func checkForMutatingWebhookConfiguration(name string) error {
-	return framework.Global.Client.Get(context.Background(), types.NamespacedName{Name: name}, &admissionregistrationv1.MutatingWebhookConfiguration{})
+	mutatingWebhookConfig := admissionregistrationv1.MutatingWebhookConfiguration{}
+	err := framework.Global.Client.Get(context.Background(), types.NamespacedName{Name: name}, &mutatingWebhookConfig)
+	if err != nil {
+		return err
+	}
+
+	err = checkRelationshipLabels(mutatingWebhookConfig.GetLabels(), "MutatingWebhookConfiguration", name)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func checkRelationshipLabels(labels map[string]string, kind, name string) error {
+	if labels == nil {
+		return nil
```

Comment:
Kept the same behavior as on master, there are cases where there is no labels at all,
see here when i tried to fix it
https://github.com/kubevirt/cluster-network-addons-operator/pull/859

i can try to understand better if it just on lifecycle and fix it, since checkRelationshipLabels
is not run on lifecycle

Reply:
I see. can you open an issue on this so we remember to further investigate?

Reply:
ok
note that In this PR i already fixed it for the relationships labels

Reply:
https://github.com/kubevirt/cluster-network-addons-operator/issues/875